bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartGraphs.java

```diff
@@ -15,6 +15,21 @@
  *******************************************************************************/
 package net.tourbook.ui.tourChart;
```

Comment:
Yes, I agree, wanted to not change the already existing pattern at first.
But here I think , I should rather use a proper "List"(instead of the Grid as now) with names beside eachcheckbox for each entry.
@wolfgang-ch  is it an issue if the Grid becomes a List (at least when you have custom tracks) ?

Reply:
This is my sugesstion:

When there are no custom tracks, keep the UI as it is, however when there are custom tracks available, display all graphs in a list which you suggested